platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/codecs/transform/IsolateTransformParametersCodec.kt

```diff
@@ -60,8 +59,7 @@ class IsolateTransformParametersCodec(
             isolatableFactory,
             buildOperationExecutor,
             classLoaderHierarchyHasher,
-            fileCollectionFactory,
-            ownerService()
```

Comment:
❔Just interested, was this also found? I don't see the connection with the PAPI.

Reply:
The `IsolateTransformParameters` constructor took a `Problems` instance.
That got removed by IDEA refactoring when I removed the parameter.